ksp_plugin/vessel.hpp

```diff
@@ -135,6 +135,7 @@ class Vessel {
   not_null<Celestial const*> parent_;
   not_null<Ephemeris<Barycentric>*> const ephemeris_;
 
+  std::unique_ptr<Part> dummy_part_;
```

Comment:
I don't quite understand this change, the previous code was nicer, with all the parts owned by the map including the dummy one.  Now we have to manage this special case in addition to the normal one.  What gives?